.editorconfig

```diff
@@ -148,7 +148,7 @@ dotnet_diagnostic.CA1823.severity = warning
 dotnet_diagnostic.CA1848.severity = none
 dotnet_diagnostic.CA1900.severity = warning
 dotnet_diagnostic.CA1903.severity = warning
-dotnet_diagnostic.CA2000.severity = warning
+dotnet_diagnostic.CA2000.severity = none
```

Comment:
REQ: Wird die Rule so nicht für alle Projekte abgeschaltet? Falls ja, muss die Rule direkt in `*.csproj`-File des Testprojekts abgeschaltet werden.